js/main.js

```diff
@@ -1,9 +1,18 @@
 // main.js
 // Contains Javascript functions.....
 
-// Easter egg
-var rawr = function() {
-  document.getElementsByClassName('container')[0].style.paddingTop = 0;
-  document.getElementsByClassName('banner')[0].style.display = "none";
-  document.getElementsByClassName('soon')[0].innerHTML = "<img height='200px' src='images/rawrburt.png'>";
-}
+"use strict";
+
+(function() {
+  var soon = document.getElementsByClassName('soon')[0];
+
+  // Easter egg
+  document.getElementById("rawr").onclick = function() {
+      soon.innerHTML = "";

```

Comment:
So I would just hide 'soon', and have a `<div>` underneath that I would populate with **rawrburt**